src/test/java/core/basesyntax/db/StorageTest.java

```diff
@@ -0,0 +1,86 @@
+package core.basesyntax.db;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.model.User;
+import core.basesyntax.service.RegistrationService;
+import core.basesyntax.service.RegistrationServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class StorageTest {
+    private final RegistrationService registrationService = new RegistrationServiceImpl();
+    private User firstUser;
+    private User secondUser;
+    private List<User> people = new ArrayList<>();
+
+    @BeforeEach
+    void setUp() {
+        firstUser = new User();
+        firstUser.setLogin("testLogin");
+        secondUser = new User();
+        secondUser.setLogin("testLogin");
+    }
+
+    @Test
+    void userWithSuchLogin() {
+        User user1 = new User();
+        user1.setLogin("User1234");
+
+        boolean userExists = false;
+        for (User user : people) {
+            if (user1.getLogin().equals(user.getLogin())) {
+                userExists = true;
+                break;
+            }
+        }
+
+        assertFalse(userExists, "User with such login already exists");
+    }
+
+    @Test
+    void userLoginLeastThan6_Ok() {
+        firstUser.setLogin("123456");
+        String login = firstUser.getLogin();
+        assertTrue(login.length() >= 6, "user's login is at least 6 characters");
+    }
+
+    @Test
+    void userPasswordLeastThan6_Ok() {
+        firstUser.setPassword("1234567");
+        String password = firstUser.getPassword();
+        assertTrue(password.length() >= 6, "user's password is at least 6 characters");
+    }
+
+    @Test
+    void userAgeLeastThan10_Ok() {
+        firstUser.setAge(19);
+        int age = firstUser.getAge();
+        assertTrue(age >= 18, "user's age is at least 18 years old");
+    }
+
+    @Test
+    void passwordIsNotEmpty() {
+        firstUser.setPassword("123456");
+        String password = firstUser.getPassword();
+        assertFalse(password.isEmpty(), "user's password should not be empty");
+    }
+
+    @Test
+    void loginIsNotEmpty() {
+        firstUser.setLogin("testLogin");
+        String login = firstUser.getLogin();
+        assertFalse(login.isEmpty(), "user's login should not be empty");
+    }
+
+    @Test
+    void ageIsNotEmpty() {
+        firstUser.setAge(null);
+        String age = firstUser.getLogin();
+        assertFalse(age.isEmpty(), "user's age should not be empty");
+    }
+
```

Comment:
```suggestion
```

Reply:
<img width="672" alt="Screenshot 2024-03-05 at 16 13 34" src="https://github.com/mate-academy/jv-registration-validation/assets/146957474/c7985512-94df-4c0a-ae37-c538fcf4a6c2">
<img width="752" alt="Screenshot 2024-03-05 at 16 16 33" src="https://github.com/mate-academy/jv-registration-validation/assets/146957474/d982e418-129a-46d3-83c9-7d35bd8c0a33">

Reply:
but i have fixed it

Reply:
and if I switch to a new line, mvn clean of the package will throw an error

Reply:
![Uploading Screenshot 2024-03-05 at 16.21.54.png…]()

Reply:
<img width="1435" alt="Screenshot 2024-03-05 at 16 21 54" src="https://github.com/mate-academy/jv-registration-validation/assets/146957474/30b78d62-e5f4-4c2e-8a3e-d4c73d8fc466">

Reply:
no, line 85 should be deleted. End of the file is correct